xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/TransactionType.java

```diff
@@ -37,6 +37,11 @@
    */
   DEPOSIT_PRE_AUTH("DepositPreauth"),
 
+  /**
+   * The {@link TransactionType} for the {@link EnableAmendment} transaction.
+   */
+  ENABLE_AMENDMENT("EnableAmendment"),
```

Comment:
Can you add Unit tests that assert the capitalization and contents of each value in this class (i.e., in `TransactionType`)?